cli/src/main.rs

```diff
@@ -11,7 +11,7 @@ use oo7::{
     zbus,
 };
 
-const BINARY_NAME: &'static str = env!("CARGO_BIN_NAME");
+const BINARY_NAME: &str = env!("CARGO_BIN_NAME");
```

Comment:
recent rust complains about this

Reply:
Removed, if needed can be done elsewhere.